### PR TITLE
User story4 edit button

### PR DESCRIPTION
**Pull request by minatimar**

### CODE REVIEW

File: components/AddDonation.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ export default function AddDonation({ createDonation }) {
             id="contactInformation"
             name="contactInformation"
             maxLength="25"
-            pattern="^[a-zA-ZäüöÄÜÖß0-9 _+&*.,:;!?()%]+"
+            pattern="^[a-zA-ZäüöÄÜÖß0-9 @_+&*.,:]+"
             // required
```
Comment (AGeck113) [inline components/AddDonation.js:84]: Just a reminder that here is "requiered" commented out. 
If that's intended it's no problem.
Edit: saw it on multiple inputs, if you want to change it

**Review by AGeck113 — APPROVED** [review]

File: components/AddDonation.js (hunk 1 above)
Comment (minatimar) [reply components/AddDonation.js:84]: this is only for testing reasons, so I or you don't have to fill out everything to see if the functions are working or not... when I'm done I will comment it in for sure :)

File: components/DonationListItem.js [hunk 2]
```diff
@@ -1,87 +1,135 @@
...
+          </StyledCard>
+        ) : (
+          <StyledCard>
+            <p>{donation.image}</p>
+            <p>Kategorie: {donation.category}</p>
+            <p>Beschreibung: {donation.description}</p>
+            <p>Haltbar bis: {donation.bestBefore}</p>
+            <p>PLZ: {donation.zipCode}</p>
+            <p>Stadtteil: Hamburg- {donation.district}</p>
+            <p>Kontakt: {donation.contactInformation}</p>
```
Comment (mbosselmann) [inline components/DonationListItem.js:42]: This part could also be good as a list. But you do not need to change that. Just an idea. ;)

File: components/AddDonation.js (hunk 1 above)
Comment (mbosselmann) [reply components/AddDonation.js:84]: I would also recommend to remove the outcommented required attributes or to not comment them out. The danger is that you'll forget to comment the commented out stuff back in later. Then add the required atrributes later. :)

**Review by mbosselmann — COMMENTED** [review]
Hi Mina,
I looked through your code and I only have two comments, but nothing you need to change.
You rock! 🪨

**Pull request merged**